src/main/resources/application.conf

```diff
@@ -51,16 +58,20 @@ backend {
     // baseExecutionBucket = ""
     // endpointUrl = ""
 
+    // [Optional]
+    // If defined, data localization and delocalization will be done on behalf of another entity (typically a user).
+    // This allows cromwell to localize data that wouldn't be accessible
+    // using whatever "authScheme" has been defined in the "google" configuration (see below).
+    // It requires a user refresh token to be passed upon every workflow submission (see "workflow options" in cromwell documentation).
+    // client_id and client_secrets must be the ones that have been used to generate the above-mentioned refresh token.
+    // localizeWithRefreshToken = {
+    //   client_id = ""
+    //   client_secret = ""
+    // }
```

Comment:
1) Yes I think this is clear.
2) Not sure the JES backend section is the best place for this ultimately, there have been rumblings that we might  support GCS paths on backends other than JES in which case this would need to be moved.
3) Don't have an opinion on this yet.  If the secrets are not passed literally of course they'd need to be at a location that's secure but could still be read by the service account.